src/main/java/org/fundacionjala/coding/manuel/Kata2.java

```diff
@@ -0,0 +1,46 @@
+package org.fundacionjala.coding.manuel;
+
+import java.util.Arrays;
+import java.util.Collections;
+
+
+/**
+ * Created by Administrator on 2/23/2018.
+ */
+public class Kata2 {
+    private String sentence;
+    private static final int MAX_LENGHT = 3;
+
+    /**
+     * Created by Administrator on 2/23/2018.
+     * @param sentence es la oracion original.
+     */
+    public Kata2(String sentence) {
+        this.sentence = sentence;
+    }
+
+    /**
+     * @return string que es la oracion ordenada.
+     */
+    public String sortInnerContent() {
+        String[] words = sentence.split(" ");
+        StringBuffer newSentece = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)